iac-extensions/docker/src/main/java/org/sonar/iac/docker/parser/grammar/DockerLexicalConstant.java

```diff
@@ -29,7 +29,9 @@ private DockerLexicalConstant() {
   public static final String COMMENT = "(?:" + LexicalConstant.SINGLE_LINE_COMMENT_HASH + ")";
   public static final String STRING_LITERAL_WITH_QUOTES = "\"(?:[^\"\\\\]*+(?:\\\\[\\s\\S])?+)*+\"";
   public static final String STRING_LITERAL_WITHOUT_QUOTES = "[^\\s=]+";
+  public static final String STRING_LITERAL_WITHOUT_QUOTES_NO_COLON = "[^\\s:]+";
```

Comment:
Do you actually use this constant? Same for `STRING_LITERAL_NO_COLON`

Reply:
No more needed indeed, thanks.